platform/ios/scripts/publish.sh

```diff
@@ -27,7 +27,7 @@ cd build/ios/pkg
 ZIP=mapbox-ios-sdk-${PUBLISH_VERSION}${PUBLISH_STYLE}.zip
 step "Compressing ${ZIP}…"
 rm -f ../${ZIP}
-zip -r ../${ZIP} *
+zip -yr ../${ZIP} *
```

Comment:
There are no symlinks in the iOS SDK, so there’s no developer impact in practice. I made this change to the iOS SDK regardless in case we ever do need a symlink in there.